### PR TITLE
Issue 3582 - V5 Card, tickets example

### LINKED ISSUE #3582: Custom tickets - v5 Card
### Description
We want to move on replacing the components inside the viewer to v5. One of the building blocks is the card.

### Goals
- [ ] As a user I want to navigate to the v5 viewer and have the the cards with the new ui 

### Tasks
- [ ] Test out a way to include a v5 card in the viewer.
- [ ] Create the card according to the design

### Notes
- This is only for the empty card without content and that can be shown in the viewer
- This card wont have the buttons from the top right
-  Figure a way to understand how this integrates in general with the viewer

### Related Resources
[MockUp](https://www.figma.com/file/VN7KvPD5hqDlTuqdIsm3vO/Custom-tickets?node-id=46%3A5533)

### PR DESCRIPTION
**Pull request by sanmont3drepo**
This fixes #3582 
#### Description
-  Added a version of the cards just for v5 components

### CODE REVIEW

File: frontend/src/v5/ui/components/viewer/cards/card.styles.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/**
```
Comment (Amantini1997) [inline frontend/src/v5/ui/components/viewer/cards/card.styles.ts:1]: I am not sure about this, but this file is the `.styles.ts` file for `tickets.tsx`, but it is named differently and located in a different directory. Is it on purpose?

File: frontend/src/v5/ui/components/viewer/cards/card.styles.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	display: flex;
+	align-items: center;
+	height: 48px;
+	padding: 0px 15px;
+    border-bottom: 1px solid  ${({ theme }) => theme.palette.base.lightest};
+	gap: 4px;
+`;
+
+export const CardContent = styled(ScrollArea)`
+	& > div {
```
Comment (Amantini1997) [inline frontend/src/v5/ui/components/viewer/cards/card.styles.ts:47]: This does not work as intended. Targeting all the direct children of the scrollbar which are `div`s results in padding added next to the scollbar. 
image

You should replace it with `& > :first-child` to fix it.

Also note that the scrollbar does not fade away when inactive

File: frontend/src/v5/ui/components/viewer/cards/card.styles.ts (hunk 1 above)
Comment (sanmont3drepo) [reply frontend/src/v5/ui/components/viewer/cards/card.styles.ts:1]: This is for cards in the viewer for v5. Is not tickets specific.

File: frontend/src/v5/ui/routes/viewer/tickets/tickets.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
+/**
```
[the remark this answers is not in the document]
Comment (sanmont3drepo) [reply frontend/src/v5/ui/routes/viewer/tickets/tickets.tsx:1]: Thats weid, I thought I did named it that way.

File: frontend/src/v5/ui/components/viewer/cards/card.styles.ts (hunk 2 above)
Comment (sanmont3drepo) [reply frontend/src/v5/ui/components/viewer/cards/card.styles.ts:47]: ahh, ok. But I think I'll create a component for that. Targeting just the first child could create problems if the content consists of more than one children

**Review by Amantini1997 — APPROVED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Review by The-Daniel — APPROVED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Pull request merged**